code/modules/ai/effect_node.dm

```diff
@@ -6,22 +6,29 @@
 	icon = 'icons/effects/landmarks_static.dmi'
 	icon_state = "x6" //Pure white 'X' with black borders
 	anchored = TRUE //No pulling those nodes yo
-	invisibility = INVISIBILITY_OBSERVER //Not visible at all
-	var/list/adjacent_nodes = list() // list of adjacent landmark nodes
-	var/list/weights = list(ENEMY_PRESENCE = 0, DANGER_SCALE = 0) //List of weights for the overall things happening at this node
+	invisibility = INVISIBILITY_OBSERVER //Visible towards ghosts
+	var/list/adjacent_nodes //list of adjacent landmark nodes
+
+	/*
+	List of weights for scoring stuff happening here; ultilizes "identifiers" to differentiate different kinds
+	of AI types looking at the same node.
+	*/
+	var/list/weights = list(
+							IDENTIFIER_XENO = list(NODE_LAST_VISITED = 0)
+						)
```

Comment:
I think this particular ")" will fail travis because it's indented too far out.

Reply:
Thankfully didn't seem to fail

Reply:
Please follow the 'intentions' of travis, for it might return one day.

make the ")" at the same indent level of `var/list/weights = list(`